lectures/markov_chains_I.md

```diff
@@ -65,87 +62,77 @@ from matplotlib.patches import Polygon
 from mpl_toolkits.mplot3d.art3d import Poly3DCollection
 ```
 
-## Definitions and examples
+## 定义与示例
 
-In this section we provide some definitions and  elementary examples.
+在本节中，我们将提供一些定义和基本示例。
 
 (finite_dp_stoch_mat)=
-### Stochastic matrices
-
-Recall that a **probability mass function** over $n$ possible outcomes is a
-nonnegative $n$-vector $p$ that sums to one.
-
-For example, $p = (0.2, 0.2, 0.6)$ is a probability mass function over $3$ outcomes.
+### 随机矩阵
 
-A **stochastic matrix** (or **Markov matrix**)  is an $n \times n$ square matrix $P$
-such that each row of $P$ is a probability mass function over $n$ outcomes.
+回想一下，**概率质量函数**是一个 $n$ 个可能结果上的非负 $n$-维向量 $p$，其总和为 1。
 
-In other words,
+例如，$p = (0.2, 0.2, 0.6)$ 是一个三个结果上的概率质量函数。
 
-1. each element of $P$ is nonnegative, and
-1. each row of $P$ sums to one
+**随机矩阵**（或**马尔科夫矩阵**）是一个 $n \times n$ 的方阵 $P$，其中 $P$ 的每一行都是一个 $n$ 个结果上的概率质量函数。
 
-If $P$ is a stochastic matrix, then so is the $k$-th power $P^k$ for all $k \in \mathbb N$.
+换句话说，
 
-You are asked to check this in {ref}`an exercise <mc1_ex_3>` below.
+1. $P$ 的每个元素都是非负的，且
+1. $P$ 的每一行的和为 1
 
+如果 $P$ 是一个随机矩阵，则对于所有 $k \in \mathbb N$，$P^k$ 也是一个随机矩阵。
 
-### Markov chains
+你将在{ref}`一个练习 <mc1_ex_3>`中验证这一点。
 
-Now we can introduce Markov chains.
+### 马尔科夫链
 
-Before defining a Markov chain rigorously, we'll  give some examples.
+现在我们可以引入马尔科夫链。
 
+在严格定义马尔科夫链之前，我们先给出一些示例。
 
 (mc_eg2)=
```

Comment:
Hi @mmcky,

I think the tag is there but I am not sure why it is not recorded. Would you mind having a look?